src/QirRuntime/test/QIR-static/qir-driver.cpp

```diff
@@ -58,7 +58,7 @@ extern "C" int64_t Microsoft__Quantum__Testing__QIR__Test_Arrays( // NOLINT
     int64_t index,
     int64_t val,
     bool dummy);
-TEST_CASE("QIR: Using 1D arrays", "[qir]")
+TEST_CASE("QIR: Using 1D arrays", "[qir][qir.arr1d]")

```

Comment:
Will catch's filters still work with category specified like this?

Reply:
For me it does work like this.
```
<test.exe> 
<test.exe> [qir]
<test.exe> [qir.arr1d]
```
And I had a test failure that was showing up on exactly one testcase (coincidentally exactly this one ;-) ). So I wanted to fix-and-run multiple times one particular testcase. So I added this finer granularity.

Reply:
FYI, to run just a single test, you can use its description string as the filter

Reply:
What does the negative filter do? (test.exe ~[qir]) Does it still filter our the whole category?

Reply:
Yes, it filters our the whole [qir] category.